content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -192,108 +193,107 @@ Install Istio and expose the control plane so that your virtual machine can acce
 
 ## Create files to transfer to the virtual machine
 
-1. Create a template `WorkloadGroup` for the VM(s)
+First, create a template `WorkloadGroup` for the VM(s):
 
-    {{< tabset category-name="registration-mode" >}}
+{{< tabset category-name="registration-mode" >}}
 
-    {{< tab name="Default" category-value="default" >}}
+{{< tab name="Default" category-value="default" >}}
 
-    {{< text bash >}}
-    $ cat <<EOF > workloadgroup.yaml
-    apiVersion: networking.istio.io/v1alpha3
-    kind: WorkloadGroup
-    metadata:
-      name: "${VM_APP}"
-      namespace: "${VM_NAMESPACE}"
-    spec:
-      metadata:
-        labels:
-          app: "${VM_APP}"
-      template:
-        serviceAccount: "${SERVICE_ACCOUNT}"
-        network: "${NETWORK}"
-    EOF
-    {{< /text >}}
-
-    {{< /tab >}}
+{{< text bash >}}
+$ cat <<EOF > workloadgroup.yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: WorkloadGroup
+metadata:
+  name: "${VM_APP}"
+  namespace: "${VM_NAMESPACE}"
+spec:
+  metadata:
+    labels:
+      app: "${VM_APP}"
+  template:
+    serviceAccount: "${SERVICE_ACCOUNT}"
+    network: "${VM_NETWORK}"
+EOF
+{{< /text >}}
 
-    {{< tab name="Automated WorkloadEntry Creation" category-value="autoreg" >}}
+{{< /tab >}}
 
-    {{< warning >}}
-    This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
-    considered `pre-alpha`.
-    {{< /warning >}}
+{{< tab name="Automated WorkloadEntry Creation" category-value="autoreg" >}}
 
-    1. Generate the `WorkloadGroup`:
+{{< warning >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `pre-alpha`.
+{{< /warning >}}
 
-    {{< text bash >}}
-    $ cat <<EOF > workloadgroup.yaml
-    apiVersion: networking.istio.io/v1alpha3
-    kind: WorkloadGroup
-    metadata:
-      name: "${VM_APP}"
-      namespace: "${VM_NAMESPACE}"
-    spec:
-      metadata:
-        labels:
-          app: "${VM_APP}"
-      template:
-        serviceAccount: "${SERVICE_ACCOUNT}"
-        network: "${NETWORK}"
-    EOF
-    {{< /text >}}
+{{< text bash >}}
+$ cat <<EOF > workloadgroup.yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: WorkloadGroup
+metadata:
+  name: "${VM_APP}"
+  namespace: "${VM_NAMESPACE}"
+spec:
+  metadata:
+    labels:
+      app: "${VM_APP}"
+  template:
+    serviceAccount: "${SERVICE_ACCOUNT}"
+    network: "${VM_NETWORK}"
+EOF
+{{< /text >}}
 
-    1. Push the `WorkloadGroup` to the cluster:
+Then, to allow automated `WorkloadEntry` creation, push the `WorkloadGroup` to the cluster:
 
-    {{< text bash >}}
-    $ kubectl --namespace ${VM_NAMESPACE} apply -f workloadgroup.yaml
-    {{< /text >}}
+{{< text bash >}}
+$ kubectl --namespace ${VM_NAMESPACE} apply -f workloadgroup.yaml
+{{< /text >}}
 
-    {{< /tab >}}
+{{< /tab >}}
 
-    {{< /tabset >}}
+{{< /tabset >}}
```

Comment:
Shouldn't these be moved above line 245 (i.e., the `Then, to allow ...` part should be after the tabset)?

Reply:
That tab should contain both steps, they're both auto-registration related.

Reply:
Instead, I've moved "First, create a template..." into each tab.